RescheduleAppointments/README.md

```diff
@@ -10,7 +10,9 @@ We recommend taking the [Quick Start: Lightning Web Components](https://trailhea
 
 ## Set Up the LWC
 
-1. Download the source code.
+1. Get the source code.
+    1. From LWC-Mobile-Samples, click Code.
```

Comment:
Please follow the same format as in [here](https://github.com/forcedotcom/LWC-Mobile-Samples/tree/main/FindNearby:

1. Make LWC-Mobile-Samples as link
2. Make code bold